### DISCUSSION TITLE
Error messages not shown?

### DISCUSSION THREAD

**Discussion by IhsenBouallegue** [prisma/prisma#15667] [Q&A]
Hello!

Is there a way to get more detailed error messages? I would love if I could get a human-readable message when an error occurs. It will simplify error handling and debugging.

This is the error I am getting now.
```json
{
  "error": {
    "code": "P2021",
    "clientVersion": "4.2.1",
    "meta": {
      "table": "public.Hub"
    }
  }
}
```

**Comment by nurul3101** [comment]
Hi @IhsenBouallegue 👋 
We usually provide an error message as well, can you please let us know how did you get this error, so we can fix this?

**Reply by IhsenBouallegue** [reply]
The error was caused by either an empty or not-matching database. I reset everything and seeded to solve the error.
I have read about the error messages in the documentation, but I have never seen them in the errors that I faced. Sometimes I would only get `{"clientVersion": "4.2.1"}`
I am using PostgreSQL as DB (if that matters). It would be hard to provide code right now as I already solved the errors. But I will keep an eye out if that happens again and try to provide a working example.
Perhaps someone else already faced something similar?

**Reply by nurul3101** [reply]
Sure, it would be helpful if you can create a [GitHub Issue](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) when you face this issue next time.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
